app/views/spree/admin/orders/_filters.html.haml

```diff
@@ -1,5 +1,5 @@
 %div{"data-hook" => "admin_orders_index_search"}
-  = form_tag false, {name: "orders_form", "ng-submit" => "fetchResults()"} do
+  = form_tag nil, {name: "orders_form", "ng-submit" => "fetchResults()"} do
```

Comment:
I don't know but I was wondering if this could work in Rails 3 as well?

Reply:
Did you try this one?

Reply:
I checked this one and it makes no difference in Rails 3.

Reply:
I opened a new pull request with this: #4763
Once that one is in master, we can rebase this pull request.